.github/workflows/binderbot.yaml

```
@@ -52,25 +55,29 @@ jobs:
       - name: Create Clean Branch
         run: |
           git checkout --orphan ${{ steps.parse.outputs.binderbot_target_branch }}
-          git rm --cached -r .
+          #git rm --cached -r .
+      - name: Debugging
+        run: |
+          echo current_directory: `pwd`
+          ls -ltRh
       - name: Build notebooks using Binderbot
         run: |
           python -m binderbot.cli \
           --binder-url ${{ steps.parse.outputs.binder_url }} \
           --repo ${{ steps.parse.outputs.binder_repo }} \
           --ref ${{ steps.parse.outputs.binder_ref }} \
           --output-dir . \
-          notebooks/Run-Anywhere/*.ipynb
+          notebooks/Run-Anywhere/*/*.ipynb
       - name: Commit files
-        if: github.ref == 'refs/heads/master'
+        if: github.ref == 'refs/heads/main'
         run: |
           git config --local user.email "action@github.com"
           git config --local user.name "GitHub Action"
           git add binder-gallery.yaml thumbnail.png *.ipynb
           git commit -m "Binderbot output"
       - name: Push commit
-        if: github.ref == 'refs/heads/master'
-        uses: ad-m/github-push-action@master
+        if: github.ref == 'refs/heads/main'
+        uses: ad-m/github-push-action@main
```

This should fix the action.

```suggestion
        uses: ad-m/github-push-action@master
```